<?xml version="1.0" encoding="UTF-8"?>
<!--
    $Author: zhangxing5129  
    $Date: 2013-9-13 下午03:21:34  
    $Revision: 1.0  
    $Purpose: 
-->
<bm:model xmlns:e="aurora.service.exception" xmlns:bm="http://www.aurora-framework.org/schema/bm" xmlns:f="aurora.database.features" alias="t1" baseTable="HLS_FIN_CALC_BUTTON_TPLT_PROC">
    <bm:fields>
        <bm:field name="button_code" databaseType="VARCHAR2" datatype="java.lang.String" physicalName="BUTTON_CODE" prompt="HLS050.HLS_FIN_CALC_BUTTON.BUTTON_CODE"/>
        <bm:field name="calc_order" databaseType="NUMBER" datatype="java.lang.Long" physicalName="CALC_ORDER" prompt="HLS050.HLS_FIN_CALC_BUTTON_PROC.CALC_ORDER"/>
        <bm:field name="procedure_name" databaseType="VARCHAR2" datatype="java.lang.String" physicalName="PROCEDURE_NAME" prompt="HLS050.HLS_FIN_CALC_BUTTON_PROC.PROCEDURE_NAME"/>
    </bm:fields>
    <bm:features>
        <f:standard-who/>
    </bm:features>
    <bm:primary-key>
        <bm:pk-field name="button_code"/>
        <bm:pk-field name="calc_order"/>
    </bm:primary-key>
    <bm:query-fields>
        <bm:query-field field="button_code" queryOperator="="/>
    </bm:query-fields>
    <e:exception-descriptor-config>
        <e:exception-descriptor exception="java.sql.SQLException" handleClass="aurora.database.SQLExceptionDescriptor">
            <e:error-message code="1" message="HLS050_IMPORT_LOAD_UNIQUE_ERROR"/>
        </e:exception-descriptor>
    </e:exception-descriptor-config>
</bm:model>
